### ISSUE TITLE
Clicking Debug > Start Debugging in the menu does not invoke the entire build process

### ISSUE THREAD

**Issue by vjekob** [Microsoft/AL#262]
There is a difference between pressing F5 on keyboard, and invoking what should (presumably) be the same function from the menu by clicking Debug > Start Debugging.

This is most obvious before the extension has been built for the first time and the .navx file is not built yet.

Steps to reproduce:

1. Create a new workspace (AL: Go!)
2. Click Debug > Start Debugging (in the menu)
at this point, you get the following error:
File c:\Users\vmadmin\Documents\AL\MyCustomReport\Default publisher_MyCustomReport_1.0.0.0.navx does not exist

The same happens if you attempt to start debugging by clicking Ctrl+Shift+D, and then pressing the > button to the left of the selected launch configuration.

It seems that the AL Language extension does not bind the same behavior to the F5 keyboard press and Start Debugging command. It may be by design, but it is confusing, because the VS Code user interface clearly indicates that F5 keyboard shortcut is assigned to the Start Debugging command, so it is reasonabl to expect that invoking the Start Debugging command should be exactly the same as pressing the F5 key.

**Comment by dennisfrostlander** [comment]
Thanks for the feedback. There were a few limitations in VSCode due to which we could not implement a consistent experience for this scenario. We will revisit our options here.

**Comment by vjekob** [comment]
Yes, I expected this to be due to how VS Code exposes its API and that binding to F5 is not exactly the same as hooking the "Start Debugging" command. But if you could do anything to make this work, it would be great as it provides consistency.

Not that I personally need it, but consistency won't hurt.
